Build error: Zone js wih @ngtools/webpack `1.10.1`

<!--
We will close this issue if you don't provide the needed information.
For feature requests, delete the form below and describe the requirements and use case.
-->

### Versions
```
@ngtools/webpack: 1.10.1
```

I am using the `@ngtools/webpack` outside of the Angular CLI, and the the latest patch version from 1.10.0 to 1.10.1. I am getting the following error;

```bash
Module not found: Error: Can't resolve 'C:\git\solutions\webpack\buildin\global.js' in 'C:\git\solutions\site\node_modules\zone.js\dist'
```

### Observed behavior
```
It looks the the module resolution or webpack context is not correct.
```

### Desired behavior
No error.


### Mention any other details that might be useful (optional)
I am thinking it might be related to this: https://github.com/angular/angular-cli/commit/595168eeb251c9928d42139424e05dcb50270b08#diff-c8ee7673e7d5e8741306839a34f0c8fc

I'll try to debug some more.

--- a/packages/@ngtools/webpack/src/paths-plugin.ts
+++ b/packages/@ngtools/webpack/src/paths-plugin.ts
@@ -25,30 +25,43 @@ export function resolveWithPaths(
     return;
   }
 
+  const originalRequest = request.request.trim();
+
+  // Relative requests are not mapped
+  if (originalRequest.startsWith('.') || originalRequest.startsWith('/')) {
+    callback(null, request);
+    return;
+  }
+
   // check if any path mapping rules are relevant
   const pathMapOptions = [];
   for (const pattern in compilerOptions.paths) {
       // can only contain zero or one
       const starIndex = pattern.indexOf('*');
       if (starIndex === -1) {
-        if (pattern === request.request) {
+        if (pattern === originalRequest) {
           pathMapOptions.push({
             partial: '',
             potentials: compilerOptions.paths[pattern]
           });
         }
+      } else if (starIndex === 0 && pattern.length === 1) {
+        pathMapOptions.push({
+          partial: originalRequest,
+          potentials: compilerOptions.paths[pattern],
+        });
       } else if (starIndex === pattern.length - 1) {
-        if (request.request.startsWith(pattern.slice(0, -1))) {
+        if (originalRequest.startsWith(pattern.slice(0, -1))) {
           pathMapOptions.push({
-            partial: request.request.slice(pattern.length - 1),
+            partial: originalRequest.slice(pattern.length - 1),
             potentials: compilerOptions.paths[pattern]
           });
         }
       } else {
         const [prefix, suffix] = pattern.split('*');
-        if (request.request.startsWith(prefix) && request.request.endsWith(suffix)) {
+        if (originalRequest.startsWith(prefix) && originalRequest.endsWith(suffix)) {
           pathMapOptions.push({
-            partial: request.request.slice(prefix.length).slice(0, -suffix.length),
+            partial: originalRequest.slice(prefix.length).slice(0, -suffix.length),
             potentials: compilerOptions.paths[pattern]
           });
         }
@@ -79,7 +92,7 @@ export function resolveWithPaths(
   }
 
   const moduleResolver = ts.resolveModuleName(
-    request.request,
+    originalRequest,
     request.contextInfo.issuer,
     compilerOptions,
     host,

--- a/tests/e2e/tests/misc/module-resolution.ts
+++ b/tests/e2e/tests/misc/module-resolution.ts
@@ -5,6 +5,13 @@ import { expectToFail } from '../../utils/utils';
 
 
 export default async function () {
+  await updateJsonFile('src/tsconfig.app.json', tsconfig => {
+    tsconfig.compilerOptions.paths = {
+      '*': [ '../node_modules/*' ],
+    };
+  });
+  await ng('build');
+
   await createDir('xyz');
   await moveFile(
     'node_modules/@angular/common',

